features/Steps/OwnershipSteps.php

```diff
@@ -42,6 +42,21 @@ public function iApproveTheOwnershipWithOwnershipId(string $ownershipId): void
         $this->theResponseBodyShouldBeValidJson();
     }
 
+    /**
+     * @When I reject the ownership with ownershipId :ownershipId
+     */
+    public function iRejectTheOwnershipWithOwnershipId(string $ownershipId): void
+    {
+        $response = $this->getHttpClient()->postJSON(
+            '/ownerships/' . $this->variableState->replaceVariables($ownershipId) . '/reject',
+            ''
```

Comment:
Why do we have to send an empty body? Seems not elegant?

Reply:
Do you mean inside requesting code here?